src/main/java/org/jenkinsci/plugins/parallel_test_executor/TestCaseCentricFormat.java

```diff
@@ -0,0 +1,21 @@
+package org.jenkinsci.plugins.parallel_test_executor;
+
+public enum TestCaseCentricFormat  {
+    DISABLED ("Disabled"),

```

Comment:
“Disabled” seems odd terminology here. IIUC this mode is Java semantics, so why not label it as such?